web/impact/impact/v1/views/reserve_office_hour_view.py

```diff
@@ -210,3 +220,33 @@ def _response(self):
             'header': self.header,
             'detail': self.detail,
             'timecard_info': self.timecard_info})
+
+    def get_calendar_data(self, counterpart_name):
+        title = self.OFFICE_HOUR_TITLE.format(counterpart_name)
+        office_hour = self.office_hour
+        if office_hour.location is None:
+            timezone = "UTC"
+            location = "MassChallenge"
+        else:
+            timezone = office_hour.location.timezone
+            location = office_hour.location
+        meeting_info = office_hour.meeting_info
+        location_info = "{location};{meeting_info}"
```

Comment:
The semicolon should only appear if the user specifies both a location and a meeting_info
Otherwise, just show the one that is specified. (if neither is specified, presumably this field is empty)

Given the following scenarios:

location_populated: office_hour.location = "MC Boston"
meeting_info_populated: meeting_info = "617-555-1212"
location_empty: office_hour.location = ""
meeting_info_empty: meeting_info = ""

we should expect these outcomes:

location_populated and meeting_info_populated =>  location_info = "MC Boston; 617-555-1212"
location_populated and meeting_info_empty => location_info = "MC Boston"
location_empty and meeting_info_populated =>  location_info = "617-555-1212"
location_empty and meeting_info_empty =>  location_info = ""